apps/dev-recruiters/src/components/modules/NewJoinPageComponent/Drag and Drop/draganddrop.tsx

```diff
@@ -0,0 +1,225 @@
+'use client';
+import { atoms } from '@devlaunchers/components/src/components';
+import axios from 'axios';
+import { ChangeEvent, useState, useRef } from 'react';
+import {
+  AllPageSection,
+  AllSection,
+  ChooseFileSection,
+  PageArrowSection,
+  PageCenterSection,
+  PageSection,
+  PageTriangleTopRightSection,
+} from './StyledDragAndDrop';
+export default function DragAndDrop({
+  filesUploaded,
+  onFilesUploaded,
+  //,uploadError
+  //  errorEncountered,
+  //  onUploadError,
+}) {
+  const [selectFiles, setSelectFiles] = useState([]);
+
+  const [uploadFiles, setUploadFiles] = useState<any>([]);
+
+  //uncommented this
+  const [showUploadModal, setShowUploadModal] = useState(false);
+  const [isUploading, setIsUploading] = useState(false); // Uploading state
+  const maxSizeInMB = 25;
+  const maxSizeInBytes = maxSizeInMB * 1024 * 1024;
+  const allowedExtensions = /(\.doc|\.pdf|\.jpg|\.jpeg|\.png)$/i;
+  const [uploadError, setUploadError] = useState('');
+  const [canButVis, setCanButVis] = useState(true);
+  const portfolioUploadformData = new FormData();
+
+  //uncommented above lines
+  const [dragActive, setDragActive] = useState<boolean>(false);
+  let uploadedids = '';
+  const fileUpload = async (inFiles) => {
+    let uploadStatus = true;
+    setIsUploading(true);
+    const response = async () => {
+      if (!allowedExtensions.exec(inFiles[0].name)) {
+        setUploadError(
+          'The file you have chosen is not a valid file type. Please upload only .doc, .pdf, .jpg, .jpeg, and .png (Max 25MB)'
+        );
+        setIsUploading(false);
+        uploadStatus = false;
+        return false;
+      }
+      // Check file size
+
+      if (inFiles[0].size > maxSizeInBytes) {
+        setUploadError(
+          'The file you have chosen is not a valid file type. Please upload only .doc, .pdf, .jpg, .jpeg, and .png (Max 25MB)'
```

Comment:
could we abstract this to a constant and reference the constant on line 44 also? That would make it easier to maintain, since it seems this is the same message as displayed on line 44.